back/db/migrate/20241120151945_create_embeddings_similarities.rb

```diff
@@ -0,0 +1,13 @@
+class CreateEmbeddingsSimilarities < ActiveRecord::Migration[7.0]
+  def change
+    enable_extension 'vector'
+
+    create_table :embeddings_similarities, id: :uuid do |t|
+      t.column :embedding, 'shared_extensions.vector(1024)'
```

Comment:
This is untested, but we need to make sure that the type is taken from the shared_extensions schema instead of the public schema.